packages/core/src/ChartContainer.js

```diff
@@ -151,12 +151,9 @@ function ChartContainer({
   const toPng = () => {
     return domToPng(chartRef.current, {
       filter: node => {
-        if (node.nodeName === 'IMG' && !node.getAttribute('src')) {
-          return false;
-        }
         const { classList } = node;
         if (classList) {
-          if (classList.contains(DOWNLOAD_ONLY_CLASSNAME)) {
+          if (classList.join().includes(DOWNLOAD_ONLY_CLASSNAME)) {
```

Comment:
```suggestion
          if (typeof node.className === 'string' && node.className.includes(DOWNLOAD_ONLY_CLASSNAME)){
```

[classList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList) doesn't support `join` unfortunately (at least in FF).